### PR TITLE
Added added code needed to build for amd64, arm32 and arm64 processors

### PR DESCRIPTION
**Pull request by colindawson**
I used [this blog post](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/) to make changes to the build.sh that should allow multi arch builds to take place.  This should allow the project to build for AMD64 as normal as well as for ARM both 32 and 64 bit.

You will need to turn on experimental features to allow the manifest command to work properly.   I watched the build and it looks like the images are created successfully. The manifest should create the final image that will be pushed.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -24,17 +24,21 @@ if [ "$EVENT_TYPE" = "cron" ]  || [ "$DEV" = "true" ]; then
     DJ_MM3_REF=$(python get_latest_ref.py mailman/django-mailman3)
     MM3_HK_REF=$(python get_latest_ref.py mailman/mailman-hyperkitty)
 
-    # Build the mailman-core image.
-    $DOCKER build -f core/Dockerfile.dev \
+	for value in amd64 arm32v7 arm64v8
```
Comment (maxking) [inline build.sh:27]:
```suggestion
	for arch in amd64 arm32v7 arm64v8
```
Can you also rename the variables used below to `$arch` instead and other loops?

**Review by maxking — CHANGES_REQUESTED** [review]
Overall, I think it would be a good change. I remember at least one person who asked me for arm images.

If you can also check whether any changes are required in the deploy script, then it would be awesome!

**Comment by maxking** [comment]
Another thing is whether it would affect downloading of images on current users? Can they still continue to use the same tag to get new images or would they have to change something?

**Comment by colindawson** [comment]
I'm not 100% sure if it will be seamless.  As this is the first time that I've attempted to do anything with docker image creation.
From what I understand, it should allow everything to carry on seamlessly as before.    The manifest means that if you try to download the image it'll check your OS and download the correct image.   The manifest takes the original tag name.
In addition people on other OS's should can work exactly the same and the manifest routes them to the correct image.

**Review by colindawson — COMMENTED** [review]
I've changed value to arch as requested, it's consistent through the whole file.

**Comment by colindawson** [comment]
I guess this is going nowhere

**Pull request closed without merging**